ui/src/app/pages/workspace/workspace-edit.tsx

```diff
@@ -876,38 +880,35 @@ export const WorkspaceEdit = fp.flow(withRouteConfigData(), withCurrentWorkspace
           populationDetails.includes(SpecificPopulationEnum.OTHER)) {
         values = {...values, otherPopulationDetails};
         constraints['otherPopulationDetails'] = {
-          presence: {
-            allowEmpty: false
-          },
           length: {
-            maximum: 100
+            minimum: 1,
+            maximum: 100,
+            ...lengthMessages('Specific Population Other')
```

Comment:
nit: this should probably be something clearer like `Other specific populations`